.github/workflows/conformance-aws-cni.yaml

```
@@ -168,7 +168,7 @@ jobs:
           aws-secret-access-key: ${{ secrets.AWS_PR_SA_KEY }}
           aws-region: ${{ env.region }}
 
-      - name: Create EKS cluster without nodegroup
+      - name: Create EKS cluster with nodegroup
```

Suggestion:
```suggestion
      - name: Create EKS cluster
```
(likewise in the other file)